plugins/jira-dashboard-backend/src/service/router.ts

```diff
@@ -106,7 +106,8 @@ export async function createRouter(
         return;
       }
 
-      const projectKey = entity.metadata.annotations?.[projectKeyAnnotation]!;
+      const projectKey =
+        entity.metadata.annotations?.[projectKeyAnnotation]?.split(',')!;
```

Comment:
There is no documentation so I don't know how this is supposed to work, but this adds support for adding multiple project keys like this? 

`jira.com/project-key: ABC,DEF`

But in the rest of the PR only first item of this array is used? What purpose serves the other keys?

Reply:
Hi [anicke](https://github.com/anicke)
In cases where multiple project keys are present, only the first project key (projectKey[0]) is used for display on the project card i.e, in this case project-key "ABC", assuming it represents the main project. However, the response table will include data from all project keys specified in the annotations.